map.js

```diff
@@ -26,20 +24,20 @@ let mousedown = false;
 let img_Map;
 let icons = {};
 
+//TODO: These all need to be reworked to just use the JsonData trees
 let locArr = [];
 let nirnArr = [];
-
-let debug = false; //makes iframe and guide small by default for map function testing.
-let discovered = false;//to see how it looks when a place is discovered, change this to true.
+let discoveredArr = []; //I know you probably hate me for adding ANOTHER array to fix later.
```

Comment:
Lol it's fine. It's better that you are getting something down than worrying about the exact specifics, which are kinda complex. I'll go ahead and create a new PR that will remove all of these .